OJP_FareSupport.xsd

```diff
@@ -238,6 +237,11 @@
 				</xs:annotation>
 			</xs:element>
 			<xs:group ref="FareAuthorityGroup"/>
+			<xs:element name="ProtoProduct" type="xs:boolean" default="false" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Is the product a proto product? If yes, it should not be shown to the user directly. Instead a proto product transports fare product information, which usually has to be processed further. Default is false.</xs:documentation>
```

Comment:
Extend by 
> Can be used for tariffing in a distributed environment, where OJP services can deliver only parts of a fare product.

?

Reply:
Shall I explain, why a proto fare product can be sensible in a distributed environment or shall we discard the "where ..." part of the sentence?